bundles/org.openhab.binding.hdpowerview/src/test/java/org/openhab/binding/hdpowerview/HDPowerViewJUnitTests.java

```diff
@@ -287,80 +286,114 @@ public void testOnlineCommunication() {
     }
 
     /**
-     * Run a series of OFFLINE tests on the JSON parsing machinery
+     * Test generic JSON shades response
      */
     @Test
-    public void testOfflineJsonParsing() {
+    public void shadeResponseIsParsedCorrectly() throws JsonParseException {
         final Gson gson = new Gson();
-
         @Nullable
         Shades shades;
-        // test generic JSON shades response
-        try {
-            @Nullable
-            String json = loadJson("shades");
-            assertNotNull(json);
-            assertNotEquals("", json);
-            shades = gson.fromJson(json, Shades.class);
-            assertNotNull(shades);
-        } catch (JsonParseException e) {
-            fail(e.getMessage());
-        }
+        @Nullable
```

Comment:
`@Nullable` not expected for local variable.
Please check all the class.

Reply:
Fixed.